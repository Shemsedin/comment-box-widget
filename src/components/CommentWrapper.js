import styled from "styled-components";

const CommentWrapper = styled.div`
  position: relative;
`;

export default CommentWrapper;
