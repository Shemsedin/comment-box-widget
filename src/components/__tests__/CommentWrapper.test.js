import React from "react";
import renderer from "react-test-renderer";
import CommentWrapper from "../CommentWrapper";

describe("CommentWrapper", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<CommentWrapper />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
