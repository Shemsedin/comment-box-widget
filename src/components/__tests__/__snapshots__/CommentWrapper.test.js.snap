exports[`CommentWrapper snapshot renders 1`] = `
<div
  className="sc-AxjAm iFRHBp" />
`;
